src/legacy/ui/public/vis/editors/default/components/default_editor_agg.tsx

```diff
@@ -0,0 +1,289 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useState, useEffect } from 'react';
+import {
+  EuiAccordion,
+  EuiToolTip,
+  EuiButtonIcon,
+  EuiText,
+  EuiTextColor,
+  EuiSpacer,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIconTip,
+  Color,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { AggConfig } from '../../../';
+import { DefaultEditorAggParams } from './default_editor_agg_params';
+import { DefaultEditorAggCommonProps } from './default_editor_agg_common_props';
+
+interface DefaultEditorAggProps extends DefaultEditorAggCommonProps {
+  agg: AggConfig;
+  aggIndex: number;
+  aggIsTooLow: boolean;
+  dragHandleProps: {} | null;
```

Comment:
Could it ever be null ? Just can't catch such a case..

Reply:
This is according to type definition:

![image](https://user-images.githubusercontent.com/11524637/61441217-d3a6a300-a94d-11e9-9955-8cd5f807d3a2.png)

`DraggableProvidedDragHandleProps` isn't exported from `@elastic/eui`, so I cannot import that type.